internal/tlsutil/generate_test.go

```diff
@@ -49,7 +49,7 @@ func Test_GenerateCaCertificate(t *testing.T) {
 	})
 	t.Run("Generate server certificate", func(t *testing.T) {
 		certSubj := "abcdefg"
-		certData, keyData, err := GenerateServerCertificate(certSubj, cert.Leaf, cert.PrivateKey, []string{"IP:127.0.0.1"})
+		certData, keyData, err := GenerateServerCertificate(certSubj, cert.Leaf, cert.PrivateKey, []string{"127.0.0.1"}, []string{"localhost"})
```

Comment:
Is there a way to verify the dns name was correctly added to the cert? Same as the IP is verified...